### ISSUE TITLE
Accessibility fix for searchbox in side bar [Docs]

### ISSUE THREAD

**Issue by jodylecompte** [crystal-lang/crystal#6921]
Hello!

I submitted a PR to Kemal's website fixing a few small accessability things and I was directed here as one of the changes I made was actually in `crystal doc` generated documentation, not a static part of the site. 

My goal is to add an aria-label to the sidebar's search box. I believe I've located the code that needs to be adjusted but I noted that in the readme it said issues should be opened prior to pull requests, so I thought it prudent to also get a second set of eyes to ensure I've found the right code.

https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/tools/doc/html/_sidebar.html

Thanks! 

EDIT: Post submit, it also occurs to me what the content of that label is would be affected by any internationalization already in place or planned for the near future.

**Comment by straight-shoota** [comment]
Sounds good.

There are no plans for internationalization of the docs generator API. And that probably won't need to be addressed anytime soon (or ever). API docs are typically only published in English anyway and software developers usually have at least a basic understanding of the language. So I don't see any need for translating the user interface.

And I don't think such simple improvements needs an issue to discuss this. But it's obviously also fine to ask first ;)
